public/redux/reducers/mapper.ts

```diff
@@ -54,6 +54,18 @@ export function getTypeFromMappings(
   } else {
     currentDataTypes[type] = [path];
   }
+
+  if (mappings.fields) {
+    Object.entries(mappings.fields).forEach(([field, value]) => {
+      currentDataTypes = getTypeFromMappings(
+        value as Mappings,
+        currentDataTypes,
+        resolvePath(path, field)
+      );
+    });
+    return currentDataTypes;
+  }
+
```

Comment:
So if I understand correctly, this is just recursively retrieving types from any nested mappings?

Reply:
Not any nested mappings, this block will parse only `fields`. In my testing index, it will parse this part
```
"fields": {
    "raw": {
         "type": "integer"
    }
}
```

Reply:
Got it, ok, makes sense.